libs/wingsdk/src/target-tf-gcp/permissions.ts

```diff
@@ -1,59 +1,42 @@
 import * as cloud from "../cloud";
 
-export function calculateBucketPermissions(ops: string[]): string[] {
+export function createBucketPermissions(ops: string[]): string[] {
```

Comment:
Whats the reason for the name change here? We use `calculateXxxPermissions` for all other targets